bridges/BugzillaBridge.php

```diff
@@ -170,7 +170,7 @@ protected function getUser($user)
 
         $url = $this->instance . '/rest/user/' . $user . '?include_fields=real_name';
         try {
-            $json = json_decode(getContents($url), true);
+            $json = $this->getJSON($url);
```

Comment:
dont call static method like this

Reply:
Ah right, making it static was a last minute change